CODE/Index.html

```diff
@@ -5,48 +5,58 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Libre+Franklin|Oswald:500" rel="stylesheet">
+    <title>Project1 SP,19, GD2</title>
 </head>
 
 <body>
-    <h1>Yes, the Octopus Is Smart as Heck. But Why?
-        It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
-        intelligent.</h1>
-    <h2>By Carl Zimmer
-        Nov. 30, 2018</h2>
+    <header>
 
-    <figure>
-        <img src="images/maxresdefault.jpg" alt=" Octopus on the sea bed">
-    </figure>
-    <figcaption>
+        <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
+        <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
+            intelligent.</h2>
+    </header>
+    <h3>By Carl Zimmer
+        Nov. 30, 2018</h3>
+    <aside class="pquote">
+        <blockquote>
+            <p>"Researchers have studied how certain animals evolved to be intelligent, among them apes, elephants,
+                dolphins and even some birds, such as crows and parrots.".</p>
+        </blockquote>
+
+    </aside>
 
-    </figcaption>
-    <p>To demonstrate how smart an octopus can be, Piero Amodio points to a YouTube video. It shows an octopus pulling
+
+
+
+    <p1>To demonstrate how smart an octopus can be, Piero Amodio points to a YouTube video. It shows an octopus pulling
```

Comment:
You use `p1` and other numbered `p` elements further below, but this is not a valid HTML element. All instances should be replaced with `p`, both opening and closing tags:
```suggestion
    <p>To demonstrate how smart an octopus can be, Piero Amodio points to a YouTube video. It shows an octopus pulling
```
I'm commenting on this specific instance, but **won't** comment on each example.